tests/translate/translate_d2a2c_vect.py

```diff
@@ -18,14 +69,48 @@ def __init__(self, grid):
         }
         self.in_vars["parameters"] = ["dord4"]
         self.out_vars = {
-            "uc": grid.x3d_domain_dict(),
-            "vc": grid.y3d_domain_dict(),
+            # "uc": grid.x3d_domain_dict(),
+            # "vc": grid.y3d_domain_dict(),
             "ua": {},
             "va": {},
-            "utc": {},
-            "vtc": {},
+            # "utc": {},
+            # "vtc": {},
```

Comment:
aren't uc and vc modified here? shouldn't we still compare them? If for some reason not, we can remove the lines. same question for utc/vtc. This may have been from me when I was debugging this stencil

Reply:
Good find! I just tried it locally and these fields validate again. This may have been leftover as an intermediate step as we were having issues. I'm reverting them.